src/components/MyAccount/Settings/SettingsInputForm.tsx

```diff
@@ -195,6 +228,7 @@ const SettingsInputForm = ({
                     isClearable
                     isClearableCallback={() => handleClearableCallback(index)}
                   />
+                  {index == 0 && <div style={{ width: "57px" }}> </div>}
```

Comment:
I'm not a fan of these magic numbers (72px on line 252). I don't have any good suggestions right now but I'll take a closer look.

Reply:
Same– there are a lot of random fixed values in the designs which prevent using spacing tokens the way I'd like to

Reply:
Right, I said I'll take another pass but I don't want to hold up pushing this out for another review. Okay to add comments post merge?